sublime_jedi.sublime-settings

```diff
@@ -11,6 +11,13 @@
     // Additional python package paths.
     "python_package_paths": [],
 
+    // When going to a definition
+    // true: Do you want this to go to where things are defined directly?
```

Comment:
Could you please rephrase the help text? 
Something like:
```
"follow_import": true, // go to definition command will follow to the place where it was defined
"follow_import": false, // go to definition command will follow to the place where it was imported
``` 
PS: i'm not the best writer too :)